website/docs/topics/feature-flags/feature-flag-best-practices.mdx

```diff
@@ -71,11 +159,11 @@ In [Principle 1](#1-enable-runtime-control), we proposed a set of architectural
 
 For client-side feature flags, SDKs should send the context to an evaluation server and receive the evaluated results. The evaluated results are then cached in memory in the client-side application, allowing quick lookups without additional network overhead. This provides the performance benefits of local evaluation while minimizing the exposure of sensitive data.
 
-This approach enhances privacy by minimizing the amount of sensitive data sent to the _Feature Flag Control Service_, reducing the risk of data exposure and potential security threats.
+<VideoContent videoUrls={["https://www.youtube.com/embed/mCXSAWzdn3I"]}/>
```

Comment:
@alvinometric do you think the video makes more sense here, or do you still have doubts?